lib/datadome/client.rb

```diff
@@ -50,10 +47,10 @@ def validate_request(data)
         end
 
       ValidationResponse.from_faraday_response(response)
-    rescue Faraday::Error::ConnectionFailed, Faraday::Error::TimeoutError => e
+    rescue Faraday::ConnectionFailed, Faraday::TimeoutError => e
```

Comment:
WDYT of constraining the faraday version to be newer than the one that introduced this changes?

Reply:
excellent idea. I'm starting to dig

Reply:
https://github.com/lostisland/faraday/commit/a6c6f1612c225335bf3bf94cf0b88e2e9a315671

Introduced in v0.9.0. I'll try to constrain the version to greater than 0.9.0.

Reply:
It seems to be already constrained .
`spec.add_runtime_dependency "faraday", ">= 0.9.2"` in the current gemspec

Reply:
we should be alright then. I'll save that modification in a dedicated commit with explanations.